### DISCUSSION TITLE
还在用绳网的这辈子有了

### DISCUSSION THREAD

**Discussion by Mirrorsing** [share121/inter-knot#3400] [其他]
因为我就是正义的新艾利都良好市民，我要把你们统统抓进近卫局接受朱鸢大姐姐的健康教育！！！
![无标题4_20210714223522](https://github.com/user-attachments/assets/a26b532f-bcc5-4a24-8d67-5b890aa6bd49)

**Comment by Mirrorsing** [comment]
好玩爱玩多玩<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" ><img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" ><img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by 3165341164** [comment]
但是绳网作为网络灰色地带是受默许的，你或许还能看到治安局的官方账号<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3d389fd2885b008e46d843c552c02bf3_5073438697399152551.png" alt="开拓者 吃瓜脸" width="75" >

**Comment by momomomofish** [comment]
谁家小孩？这年头上网门槛这么低了吗...当年为了上绳网搞了好久来着，现在绳网上真的是什么人都有...
赶紧@一下管理吧

**Reply by Mirrorsing** [reply]
噫……大叔好讨厌！！！<img src="https://bbs-static.miyoushe.com/static/2023/11/16/1e538aa10db4b57531cecc25b5ab3d01_7718170800642277902.png" alt="珂蕾妲-再说一遍？！" width="75" ><img src="https://bbs-static.miyoushe.com/static/2023/11/16/1e538aa10db4b57531cecc25b5ab3d01_7718170800642277902.png" alt="珂蕾妲-再说一遍？！" width="75" ><img src="https://bbs-static.miyoushe.com/static/2023/11/16/1e538aa10db4b57531cecc25b5ab3d01_7718170800642277902.png" alt="珂蕾妲-再说一遍？！" width="75" >
